code/src/Components/Details.jsx

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { Link } from "react-router-dom";
+import styled from 'styled-components'
+import arrow from '../assets/arrow.png'
+import Error from "./Error";
+
+const Details = ({films}) => {
+    const {id} = useParams()
+    const movie = films.filter(movie => movie.id === Number(id))[0];
+
+    const backgroundImageUrl = `https://image.tmdb.org/t/p/original${movie?.backdrop_path}`
+    
+    if (movie) {
+    return(
+        // <section style={backgroundStyle}>
+        <Section backgroundimage={backgroundImageUrl}>
```

Comment:
We like how you approached this solution of adding a URL to a styled component tag. The logic seem so obvious and once again gives us a 'Aha'- moment.